raiden/utils/nursery.py

```diff
@@ -87,19 +87,18 @@ def subprocess_stopped(result: AsyncResult) -> None:
                     if janitor._stop.is_set():
                         return
```

Comment:
This line has to be updated now that we return `Popen`.

Reply:
humm, it has to return `None`, since the process can not be started (it is a race condition where the python process is shutting down while the code is trying to start a new process)

Reply:
It could also raise an exception instead. But returning `None` and declaring the return value as `Optional` as you've done works, too.